packages/contracts/CHANGELOG.md

```diff
@@ -9,6 +9,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+### Changed
+
+- Added check to `execute()` in `DAO` to prevent griefing attacks if the caller provides insufficient gas on actions being allowed to fail.
+
+### Removed
+
+## v1.0.0
```

Comment:
Correct @mathewmeconry ?

Reply:
A little context would be nice but we already released v1.0.0 so I would say no? :sweat_smile:

Reply:
I am confused with the version numbers and was wondering if the changelog wasn't updated.
Is the last contracts/`@aragon/osx` version 0.7.0-alpha ? 
The NPM package is at 1.0.1 already.

Reply:
yeah then it is correct. But check that some of these entries aren't already new and not yet on mainnet